experimental/segmenter/src/complex.rs

```diff
@@ -103,41 +104,40 @@ pub fn complex_language_segment_utf16(
 pub fn complex_language_segment_str(
     dictionary: &Dictionary,
     lstm: &LstmPayloads,
+    grapheme: Option<&GraphemeClusterSegmenter>,
     input: &str,
 ) -> Vec<usize> {
     let mut result: Vec<usize> = Vec::new();
     let lang_iter = LanguageIterator::new(input);
     let mut offset = 0;
     for str_per_lang in lang_iter {
-        #[cfg(feature = "lstm")]
-        {
-            if let Some(model) = lstm.best(str_per_lang.chars().next().unwrap() as u32) {
-                if let Ok(segmenter) = LstmSegmenter::try_new_unstable(model) {
-                    let breaks = segmenter.segment_str(&str_per_lang);
-                    let mut r: Vec<usize> = breaks.map(|n| offset + n).collect();
-                    result.append(&mut r);
-                    offset += str_per_lang.chars().fold(0, |n, c| n + c.len_utf8());
```

Comment:
`optional`: I think you can do:

```rust
offset += str_per_lang.chars().map(|c| c.len_utf8()).sum();
```

playground: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=2dd47bb98d7b9b87efecdbb56a1afa11